.gitignore

```diff
@@ -16,3 +16,4 @@ bin/
 lib/
 pyvenv.cfg
 src/
+.vscode/
```

Comment:
in https://github.com/datastax/cassandra-dtest/pull/78/commits/5993b31e0500ec74182679e95416be8864e74c03 

@k-rus , do you object if I upstream this commit ?

Reply:
This is good to have, no objections :)

Reply:
done in https://github.com/apache/cassandra-dtest/commit/df208d04f6cb29817446086b1d5a79f59cd6ae6c

commit deleted.
updates in `mck/main-rebase-20250430-clean0` branch until all current comments/questions are addressed.